src/main/flight/mixer_tricopter.c

```diff
@@ -536,12 +537,26 @@ STATIC_UNIT_TESTED void tailTuneModeThrustTorque(thrustTorque_t *pTT, const bool
         }
         else
         {
+            if (IsDelayElapsed_ms(pTT->lastAdjTime_ms, 1000))
+            {
+                // There has not been any valid samples in 1 s, try to loosen the criteria a little
+                pTT->tailTuneGyroLimit += 0.1f;
+                pTT->lastAdjTime_ms = millis();
+                if (pTT->tailTuneGyroLimit > 7.0f)
+                {
+                    // Time to get here without any good samples is 35 s, with 50% of the time using tailTuneGyroLimit >5.25 deg/s.
+                    // If there are not enough samples by now it is a fail.
+                    pTT->state = TT_FAIL;
+                }
+            }
             pTT->timestamp_ms = millis();
         }
         break;
     case TT_WAIT_FOR_DISARM:
         if (!ARMING_FLAG(ARMED))
         {
+            currentProfile->servoConf[6].max = pTT->servoAvgAngle.sum / pTT->servoAvgAngle.numOf; // uncomment for debugging
```

Comment:
easy way to see what values I end up with.
Probably only needed for a few tests.